fdp/.htaccess

```diff
@@ -1,5 +1,5 @@
 RewriteEngine on
 
 # Applications
-RewriteRule ^app$ https://app.fairdatapoint.org [R=302,L]
-RewriteRule ^home$ https://home.fairdatapoint.org [R=302,L]
+RewriteRule ^app/(.*)$ https://app.fairdatapoint.org/$1 [R=307,L]
+RewriteRule ^home/(.*)$ https://home.fairdatapoint.org/$1 [R=307,L]
```

Comment:
You might consider making the `/` optional so `https://w3id.org/fdp/app` works too.
```apache
RewriteRule ^app/?(.*)$ https://home.fairdatapoint.org/$1 [R=307,L]
```

Reply:
Thanks David, will do.